wagtail/admin/templates/wagtailadmin/workflows/edit.html

```diff
@@ -59,7 +58,12 @@
                                 <ul>
                                     {% if can_enable %}
                                         <li>
-                                            <button type="submit" class="button no" data-enable-action>{{ view.enable_item_label }}</button>
+                                            <button class="button no" 
+                                                data-action="w-action#post"
+                                                data-controller="w-action"
+                                                data-w-action-url-value = '{{ view.get_enable_url }}'
```

Comment:
Just for the sake of consistency,

```suggestion
                                                data-w-action-url-value="{{ view.get_enable_url }}"
```